app/models/user.rb

```diff
@@ -21,4 +21,12 @@ def full_name
   def proper_name
     "#{last_name}, #{first_name}"
   end
+
+  def admin_in?(roster)
+    memberships.find_by(roster: roster).admin?
+  end
+
+  def admin?
```

Comment:
One alternative is having a regular old 'administrator' column on a user itself, and then allow current administrators to edit the users to make them admins.. but.. this seems kind of awful, so carry on.